error_reporting/main_test.py

```diff
@@ -17,7 +17,9 @@
 import main
 
 
-@mock.patch("fluent.event")
-def test_error_sends(event_mock):
+@mock.patch("main.error_reporting")
+def test_error_sends(error_reporting_mock):
+    client_mock = mock.Mock()
```

Comment:
Not sure exactly what you mean? Without mocks there's not much to assert, seemed better to throw in one mock to assert something.